doc/snippets/Microsoft.Data.SqlClient/EnclavePublicKey.xml

```diff
@@ -0,0 +1,14 @@
+<docs>
```

Comment:
This documentation is currently not used. But maybe we can worry about the documentation part later.

Reply:
nvm looks like EnclavePublicKey is internal. You can just remove this xml file.

Reply:
Yes, we need to remove this file.